ikvmc/Compiler.cs

```diff
@@ -952,6 +969,26 @@ private static int ResolveReferences(List<CompilerOptions> targets)
 				next_reference: ;
 				}
 			}
+			if (target.unresolvedNetStandardReferences != null)
+			{
+				foreach (string reference in target.unresolvedNetStandardReferences)
+				{
+					foreach (CompilerOptions peer in targets)
+					{
+						if (peer.assembly.Equals(reference, StringComparison.InvariantCultureIgnoreCase))
+						{
+							ArrayAppend(ref target.peerReferences, peer.assembly);
+							goto next_reference;
```

Comment:
You may remove ugly `goto` and get code more readable if you rewrite code with some LINQ.

Reply:
If I understood your code right, consider something like:

```csharp
var peerAssemblies = targets
	.Where(peer => target.unresolvedNetStandardReferences.Contains(peer.assembly,  StringComparison.InvariantCultureIgnoreCase))
	.ToArray();
foreach (var reference in peerAssemblies)
{
	ArrayAppend(ref target.peerReferences, peer.assembly);
}

foreach (var reference in target.unresolvedNetStandardReferences.Where(reference => targets.All(peer => !peer.assembly.Equals(reference, StringComparison.InvariantCultureIgnoreCase))))
{
	int rc = resolver.ResolveReference(cache, ref target.references, reference, true);
	if (rc != 0)
	{
		return rc;
	}
}
```